packages/webapp/pages/onboarding.tsx

```diff
@@ -166,6 +171,15 @@ export function OnboardPage(): ReactElement {
 
   const onClickCreateFeed = () => {
     setShouldEnrollInReadingReminder(true);
+
+    const onboardingChecklist = getFeatureValue(feature.onboardingChecklist);
+
+    if (onboardingChecklist) {
+      setSettings({
+        sidebarExpanded: true,
+        onboardingChecklistView: ChecklistViewState.Open,
+      });
+    }
```

Comment:
Basically if in experiment user will have sidebar open after onboarding and see the checklist.